src/View/ImageLoader.py

```diff
@@ -81,75 +85,82 @@ def load(self, interrupt_flag, progress_callback):
                 return False
 
             progress_callback.emit(("Getting contour data...", 30))
-            dict_raw_contour_data, dict_numpoints = \
-                ImageLoading.get_raw_contour_data(dataset_rtss)
+            dict_raw_contour_data, dict_numpoints = ImageLoading.get_raw_contour_data(
+                dataset_rtss
+            )
 
             # Determine which ROIs are one slice thick
             dict_thickness = ImageLoading.get_thickness_dict(
-                dataset_rtss, read_data_dict)
+                dataset_rtss, read_data_dict
+            )
 
             if interrupt_flag.is_set():  # Stop loading.
                 return False
 
             progress_callback.emit(("Getting pixel LUTs...", 50))
             dict_pixluts = ImageLoading.get_pixluts(read_data_dict)
-
+            progress_callback.emit(("Getting pixel LUTS...", 99))
```

Comment:
provide better user feedback, although at this point, this happens pretty quickly.